dpgen2/entrypoint/submit.py

```diff
@@ -694,10 +694,8 @@ def resubmit_concurrent_learning(
     if reuse is None:
         return None
     reuse_idx = expand_idx(reuse)
-    reuse_step = []
-    old_wf_info = old_wf.query()
-    for ii in reuse_idx:
-        reuse_step += old_wf_info.get_step(key=all_step_keys[ii])
+    reuse_keys = [all_step_keys[ii] for ii in reuse_idx]
+    reuse_step = wf.query_step_by_key(reuse_keys)
```

Comment:
```suggestion
    reuse_step = old_wf.query_step_by_key(reuse_keys)
```

Reply:
sry the changes are not intended to be committed. .